src/defaultsettings.cpp

```diff
@@ -208,11 +207,7 @@ void set_default_settings()
 	settings->setDefault("texture_path", "");
 	settings->setDefault("shader_path", "");
 #if ENABLE_GLES
-#ifdef _IRR_COMPILE_WITH_OGLES1_
-	settings->setDefault("video_driver", "ogles1");
-#else
 	settings->setDefault("video_driver", "ogles2");
```

Comment:
worth nothing that this is a relevant difference for Android, which still defaults to ogles1
someone should test this

Reply:
Worth *noting*, I suppose?
If Minetest can fallback GLES1→GLES2 reliably then always using the former as the default may be better indeed.

Reply:
It can't fallback easily.
IIRC GLES 2.x is an install requirement for the app, we just don't know how well it works.

Reply:
#12391

Switching to OGLES2 *should* be mostly fine, my NodeCore Android app already uses OGLES2 by default (w/ shaders enabled) and there haven't been any issues from it. There's obscure issues like #12390 but probably wouldn't be something players would randomly stumble upon.

Reply:
Am I right Android is the only platform using GLES1 as the default? It would be a matter of `#ifdef __ANDROID__` then.